.github/workflows/update_repo_datasets.yml

```diff
@@ -1,11 +1,14 @@
 name: Update combined datasets
 
 on:
-  # This hook is automatically triggered by
+  # This workflow is automatically triggered by
   # https://github.com/covid-projections/covid-data-public/blob/master/.github/workflows/update_data.yml
   # after covid-data-public datasets have been updated.
-  repository_dispatch:
-    types: update-dataset-snapshot
+  workflow_dispatch:
+    inputs:
+      trigger_api_build:
+        description: 'If "true" API Build will be triggered after dataset update.'
```

Comment:
nit: It took me a minute to figure out that "API Build" meant "snapshot build".  Might be worth trying to unify our internal terminology.  As a compromise we could kinda' combine terminology and say 'If "true" a new API snapshot build will be triggered ..."  

I don't care super strongly though.

Reply:
I like API snapshot build I think, will change it!